ossm-release-notes/ossm-release-notes-support-tables-assembly.adoc

```diff
@@ -0,0 +1,56 @@
+:_mod-docs_content-type: ASSEMBLY
+[id="ossm-release-notes-support-tables-assembly"]
+= {SMProductShortName} {SMProductVersion} feature support tables
+include::_attributes/common-attributes.adoc[]
+:context: ossm-release-notes-support-tables-assembly
+
+toc::[]
+
+These tables provide guidance on supported versions and feature availability in {SMProduct} 3.
+
+// Copied from OCP core file: making-open-source-more-inclusive.adoc.
+
+include::modules/ossm-release-notes-making-open-source-inclusive.adoc[leveloffset=+1]
+
+[id="release-notes-definitions"]
+== Definitions
+
+For {SMProductName} 3, features that are Generally Available (GA) are fully supported and are suitable for production use.
+
+Technology Preview (TP) features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. See the link:https://access.redhat.com/support/offerings/techpreview[Technology Preview scope of support on the Red Hat Customer Portal] for more information about TP features.
```

Comment:
I don't think we're supposed to use "TP" as an abbreviation in regular text. (In table as a designator would be fine)

```suggestion
Technology Preview (TP) features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process. See the link:https://access.redhat.com/support/offerings/techpreview[Technology Preview scope of support on the Red Hat Customer Portal] for more information about Technology Preview features.
```